specification/security/resource-manager/Microsoft.Security/preview/2023-05-01-preview/securityStandards.json

```diff
@@ -0,0 +1,385 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Defender for Cloud",
+    "description": "API spec for Microsoft.Security (Microsoft Defender for Cloud) resource provider",
+    "version": "2023-05-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/{scope}/providers/Microsoft.Security/securityStandards": {
+      "get": {
+        "x-ms-examples": {
+          "List security standards by subscription scope": {
+            "$ref": "./examples/SecurityStandards/ListBySubscriptionSecurityStandards_example.json"
+          },
+          "List security standards by security connector scope": {
+            "$ref": "./examples/SecurityStandards/ListBySecurityConnectorSecurityStandards_example.json"
+          },
```

Comment:
When the scope is Security Connector, the resource id is becoming invalid here, due to /providers/Microsoft.Security segment appearing twice. I would suggest to list resource type separately for all scopes to ensure all the explicitly valid, instead of using {scope}.

Reply:
+1, as long as this resource type is only extending one or a small number of resource types.

Reply:
Not sure why this is a problem, a proxy resource can be extension to another resource. In this case the security standard is an extension to the security connector.
I see live examples in the manifest for this behavior in governancerules.

Reply:
@talco773, I think it's the last unresolved comment for this PR. After confirming **/providers/Microsoft.Security segment appearing twice** is valid, it can be signed off. 
Could you kindly provide the link to the live examples you see before removing "ARMChangeRequest" label ?

Reply:
Hey you can see such behavior in the governance rule resource:
https://learn.microsoft.com/en-us/rest/api/defenderforcloud/governance-rules/create-or-update?tabs=HTTP#:~:text=subscriptionId%7D%27)%2C%20or%20security-,connector,-(format%3A%20%27subscriptions/%7BsubscriptionId

And also in policy assignment:
https://learn.microsoft.com/en-us/rest/api/policy/policy-assignments/create?tabs=HTTP#:~:text=providers/%7BresourceProviderNamespace%7D

Reply:
Sharing my understanding here:

Should this new resource type "securityStandards" be a child proxy resource under security connectors scope or an extension resource under security connectors scope?

Basically,
1. child proxy resource as:
`"/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/gcpResourceGroup/providers/Microsoft.Security/securityConnectors/gcpconnector/securityStandards/1f3afdf9-d0c9-4c3d-847f-89da613e70a8"`

vs.

2. extension proxy resource as:
`"subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/gcpResourceGroup/providers/Microsoft.Security/securityConnectors/gcpconnector/providers/Microsoft.Security/securityStandards/1f3afdf9-d0c9-4c3d-847f-89da613e70a8"`

Can this securityStandards resource type be extended to some other resource types (in addition to security connectors) also? Because if it is only point 1, not sure why it should be defined as extension resource?

Reply:
Security Standard can be extension only for security connector for now (it can change in the future).
But it can also be a standalone resource under subscription/management group.
For these two reasons who chose it to be extension and not child resource